File: csharp/beecrowd1002.cs
using System;
using System.Globalization;

class Programa
{
    static void Main(string[] args)
    {

        CultureInfo CI = CultureInfo.InvariantCulture;
        double area, raio;

        raio = double.Parse(Console.ReadLine(), CI);

        area = Math.Pow(raio, 2) * 3.14159;

        Console.WriteLine("A=" + area.ToString("F4", CI));
    }
}

File: c/beecrowd1002.c
#include <stdio.h>
#include <math.h>

// pi = M_PI

int main() {

    double area, raio;

    scanf("%lf", &raio);

    area = pow(raio, 2) * 3.14159;

    printf("A=%.4lf\n", area);

    return 0;
}
